test/unit/Controllers/UserController.test.ts

```diff
@@ -4,10 +4,13 @@ import {Request, Response} from "express";
 import {UserService} from "../../../src/Services/UserService";
 import {UserController} from "../../../src/Controller/Api/UserController";
 import logger from "../../../src/Services/Logger";
+import {NodeService} from "../../../src/Services/NodeService";
+import {Node} from "../../../src/Models/Node";
 
 describe("UserController", function () {
 
     const userServiceStub = sinon.createStubInstance(UserService);
+    const nodeServiceStub = sinon.createStubInstance(NodeService);
```

Comment:
Node service and Node are not needed here, it's my fault  that ```describe('GET /node/user', () => {...}``` is left here, removed it last week, somehow it popped up again. Could you please also remove that ```describe...``` code?